Gemfile

```diff
@@ -25,7 +25,9 @@ gem 'ronin-db-activerecord', '~> 0.2', github: 'ronin-rb/ronin-db-activerecord',
 
 gem 'ronin-payloads',   '~> 0.1', github: 'ronin-rb/ronin-payloads'
 # gem 'ronin-exploits',   '~> 1.0', github: 'ronin-rb/ronin-exploits'
-# gem 'ronin-vulns',      '~> 0.1', github: 'ronin-rb/ronin-vulns'
+gem 'ronin-vulns',      '~> 0.2', github: 'ronin-rb/ronin-vulns',
+                                  branch: '0.2.0'
+gem 'ronin-web',        '~> 1.0', github: 'ronin-rb/ronin-web'
```

Comment:
Not sure how this got in there. I removed `ronin-web` as a dependency recently, since it only contains CLI stuff now.